integrations/langchain_callback.py

```
@@ -109,19 +127,19 @@ def _compress_message_list(
+            f"({result.tokens_saved_pct:.1f}% saved)"
         )
 
         # Rebuild messages
         compressed_content = (
             f"[SuperCompress: {result.original_tokens}→{result.kept_tokens} tok, "
-            f"{result.tokens_saved_pct:.1f}%% saved]\n\n"
+            f"{result.tokens_saved_pct:.1f}% saved]\n\n"
             f"{result.compressed_text}\n\n---\n\n{query}"
         )
 
         return system_msgs + [HumanMessage(content=compressed_content)]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)